tools/jsrun.py

```diff
@@ -33,7 +33,13 @@ def make_command(filename, engine=None, args=[]):
   # label a path to nodejs containing a 'd8' as spidermonkey instead.
   jsengine = os.path.split(engine[0])[-1]
   # Use "'d8' in" because the name can vary, e.g. d8_g, d8, etc.
-  return engine + [filename] + (['--expose-wasm', '--'] if 'd8' in jsengine or 'jsc' in jsengine else []) + args
+  is_d8 = 'd8' in jsengine
```

Comment:
I thought about something like that but I prefer my way because it distinguishes the purposes of the 2 flags. If the no-async flag were going to go away entirely then I probably would have done something similar because the d8 and jsc cases would just collapse back together, but it looks like there's going to be a different d8 flag to enable some sort of event loop mode to get the behavior we want, so there will always be a flag there.